.cargo/audit.toml

```diff
@@ -9,4 +9,8 @@ ignore = [
                        # This is a transitive depependency
   "RUSTSEC-2021-0139", # This is about `ansi_term` not being maintained anymore
```

Comment:
I think we can drop this, cargo.lock doesn't contain `ansi_term` anymore, and now has `nu_ansi_term`.

Reply:
Good catch. Yes, we can drop that